### ISSUE TITLE
Header tag errors (most pages)

### ISSUE THREAD

**Issue by ToniBonittoGSA** [GSA/digitalgov-pra#208]
There is a navigational and accessibility problem with inconsistent coding of subheaders.

All pages correctly start with an H1 for the page title, but most skip the H2 tag to use the smaller H3 or H4 tag for sub-headers. 

We cannot skip header tags to force the formatting of text; having them in numerical order has always been required as an HTML best practice, and is best for content accessibility, (particularly for users who rely on screen readers). 

We must apply styles to the headers so that the font property’s attributes — family (e.g., Public Sans, Arial, Times New Roman, etc.), size, weight (e.g., bold), and others — display as desired on the page. 

**Header tags & sidebar nav are correct:** 
On [Get help with the PRA](https://federalist-proxy.app.cloud.gov/preview/gsa/digitalgov-pra/bh-digital-banner/contact/), H1, H2s, and H3s are coded correctly, AND the H2s do end up in the side nav! (how all pages should be)

Inconsistent use of H2s and H3s is also causing a sidebar navigation issue in the site: H3 subheaders end up in the left sidebar navigation, but most of the H2 subheaders do not. 

Some examples that need to be fixed: 

**A**: On [Do I need clearance?](https://federalist-proxy.app.cloud.gov/preview/gsa/digitalgov-pra/bh-digital-banner/do-i-need-clearance/), headers incorrectly go from H1 to two H3s. The H3s — “Who are you collecting information from?” and “What type of information are you collecting?” — appear in the sidebar navigation. 
To fix, the H3s must be changed to H2s, and make sure that they still appear in the sidebar nav. 

**B**: On the [Additional resources](https://federalist-proxy.app.cloud.gov/preview/gsa/digitalgov-pra/bh-digital-banner/additional-resources/) page, the header tags are correctly coded, going from H1 for the title and H2s for all the main subheaders, **_but_** the H2s — _PRA Systems_, _OMB Guidance Memoranda_, _PRA Full Text_, _Reports_, _Tools_ — are missing from the sidebar nav. 
It will be fixed when the H2s appear in the sidebar nav.

**C**: On the [PRA and the Web](https://federalist-proxy.app.cloud.gov/preview/gsa/digitalgov-pra/bh-digital-banner/do-i-need-clearance/pra-and-the-internet/) page, H4s are used instead of H2s. 
It will be fixed when the H2s appear in the sidebar nav.

**Comment by amandacostello** [comment]
All heading tags have been corrected in currently-open pull requests.
